.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,63 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+    - cron: '23 1 * * *'
+  # Run on demand with workflow dispatch
+  workflow_dispatch:
+
+jobs:
+  upload_nightly_wheels:
+    name: Upload nightly wheels to Anaconda Cloud
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'matplotlib'
+
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      # c.f. https://github.com/actions/download-artifact/issues/3#issuecomment-1017141067
+      - name: Download wheel artifacts from last build on 'main'
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PROJECT_REPO="matplotlib/matplotlib"
+          BRANCH="main"
+          WORKFLOW_NAME="cibuildwheel.yml"
+          ARTIFACT_NAME="wheels"
+
+          gh run --repo "${PROJECT_REPO}" \
+             list --branch "${BRANCH}" \
+                  --workflow "${WORKFLOW_NAME}" \
+                  --json event,status,databaseId > runs.json
+          # Filter on 'push' events to main (merged PRs) that have completed
+          echo "$(cat runs.json | \
+            jq -c '[ .[] | select(.event == "push") | select(.status == "completed") ]'
```

Comment:
This line is now at 87. So still not at 80, but I think this is pretty close while still keeping readability.